_content/setup-wallets-commandline.md

```diff
@@ -0,0 +1,141 @@
+## Interledger Protocol (ILP) TestNet: Getting Started
+
+This tutorial describes how to use the command line and test XRP to:
+
+1. Create an account in the Xpring.io Interledger Testnet.
+2. Add test XRP to your account using the TestNet Rainmaker \(our version of a "faucet"\).
+3. Check your account balance.
+4. Send test XRP to another payment pointer.
+5. Receive test XRP.
+
+To do the same with a GUI-based interface, refer to [setup-wallets.md].
+
+### 1. Create an account
+
+Create a new account with this command:
+
+```
+> curl --location --request POST 'https://hermes-rest.ilpv4.dev/accounts' \
+--header 'Content-Type: application/json' \
+--header 'Accept: application/json' \
+--data-raw '{
+  "assetCode": "XRP",
+  "assetScale": "6"
+}'
+```
+
+This request returns a payload that contains your `accountId`, an auth token which you will then use to authenticate to the connector, as well as a [Payment Pointer](https://paymentpointers.org/) that can be used to receive value in your test account.
+
+Here's an example response payload for reference. Keep track of the generated `accountId` at `accountId`, and your generated auth token at `"customSettings" -> "ilpOverHttp.incoming.simple.auth_token", for use in subsequent steps.
```

Comment:
"customSettings" -> "ilpOverHttp.incoming.simple.auth_token"
Any objection to styling this as: `customSettings.ilpOverHttp.incoming.simple.auth_token`  ?

Reply:
I think we should keep this notation as-is (i.e., maintain the arrow).